docker/packager/binary/Dockerfile

```diff
@@ -91,6 +91,9 @@ ENV PATH="$PATH:/usr/local/go/bin"
 ENV GOPATH=/workdir/go
 ENV GOCACHE=/workdir/
 
+RUN curl https://raw.githubusercontent.com/matus-chochlik/ctcache/1f6ce6b0581b00cffcd1f6642c7fd41cc81f2d21/clang-tidy-cache -Lo /usr/bin/clang-tidy-cache \
```

Comment:
I found no official releases for clang-tidy-cache: https://github.com/matus-chochlik/ctcache/releases

Just wondering how you found this binary?

Reply:
It's not the binary, it's a python script from a particular commit (current master)

Reply:
Oh, no. This one is from https://github.com/Felixoid/ctcache/tree/all_merged